app/src/main/java/org/simple/clinic/illustration/IllustrationSync.kt

```diff
@@ -33,6 +33,9 @@ class IllustrationSync @Inject constructor(
   override fun syncConfig() = configProvider
 
   private fun saveIllustration(illustration: HomescreenIllustration) {
+    val illustrationsFile = File(illustrationsFolder, illustration.eventId)
+    if (illustrationsFile.exists()) return
```

Comment:
Even if the file exists, we should continue to save it because a new image might have been uploaded for the same event.

Reply:
Done